lib/luanetfilter.c

```diff
@@ -43,6 +73,67 @@ static void luanetfilter_release(void *private)
 	lunatik_putobject(luaxtable->runtime);
 }
 
+/* Register the target with the netfilter module. The caller must push the table on the stack.
+ * Pops the table from the stack both on success and failure.
+ * Pushes the error string on the stack on failure and returns -1.
+ * */
+static int handle_xtable_reg(lua_State *L, luanetfilter_xtable_t *luaxtable, LUANETFILTER_MODE mode)
```

Comment:
let's prefix everything with `luanetfilter_`, e.g., `luanetfilter_xtable_register`

Reply:
Got it